package.json

```diff
@@ -53,6 +53,7 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.3.4",
     "redux": "^3.0.4",
+    "redux-devtools": "^2.1.5",
```

Comment:
I rely on redux-devtools in the tests to ensure things work as expected, otherwise `reset` is difficult to test (https://github.com/kjbekkelund/redux-simple-router/commit/4091b1e43f9a0b83243a09592aab2b5f22e239a7#diff-413f2628e9c93c46cec6db202ddb8ea4R165). If we only dispatch the initial state as payload it will become a new object and won't be `=== initialState`.